src/services/storage/StorageManager.ts

```
@@ -111,6 +114,125 @@ export class StorageServiceManager {
+        connectionData = {
+          url: ossUrl,
+          username: config.username!,
+          password: config.password!,
+          connected: true,
+          metadata: {
+            bucket: config.bucket,
+            region: config.region,
+            endpoint: config.endpoint || config.url
+          }
+        };
+        await connectionStorage.saveConnection(connectionData, config.name, true);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)